system/metrics/sinks/prometheus/prometheusSink.hpp

```diff
@@ -75,28 +67,22 @@ class PROMETHEUSSINK_API PrometheusMetricSink : public MetricSink
     StringBuffer                    m_metricsServiceName;
     bool                            m_verbose;
 
+    static constexpr const char * PROMETHEUS_REPORTER_TYPE = "prometheus";
     static constexpr const char * PROMETHEUS_METRICS_HTTP_PAGE_TITLE = "HPCC Systems - Prometheus Metrics Service";
     static constexpr const char * PROMETHEUS_METRICS_SERVICE_RESP_TYPE = "text/html; charset=UTF-8";
     static constexpr int          DEFAULT_PROMETHEUS_METRICS_SERVICE_PORT = 8767;
-    static constexpr bool         DEFAULT_PROMETHEUS_METRICS_SERVICE_SSL = false;
     static constexpr const char * DEFAULT_PROMETHEUS_METRICS_SERVICE_NAME = "/metrics";
     static constexpr const char * HTTPLIB_ERROR_MESSAGE_HEADER_NAME = "EXCEPTION_WHAT";
-    static constexpr const char * PROMETHEUS_METRICS_HTTP_ERROR = "\
-        <html>\
-        \
-        <head>\
-        <meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\">\
-        <title>HPCC Systems - Prometheus Metrics Service</title>\
-        </head>\
-        \
-        <body>\
-        <H1>Error: %s</H1>\
-        <H2>Path: %s</H2>\
-        <H2>Status: %d</H2>\
-        </body>\
-        \
-        </html>\
-        ";
+    static constexpr const char * PROMETHEUS_METRICS_HTTP_ERROR = R"!!(
+<html><head>
+<meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\">
```

Comment:
Have you tested this?  I don't think you want the \ in front of the "s.  Also, will it be a problem having a newline before \< html >?

Reply:
I passed my eye-ball test: 
![image](https://user-images.githubusercontent.com/915949/121061427-d6088500-c791-11eb-9680-f97da0a86247.png)
but you're right, I'll update the string.